resources/puppetlabs/lein-ezbake/template/pe/ext/redhat/init.suse.erb

```diff
@@ -104,39 +92,20 @@ start() {
     fi
     <% end -%>
 
-    # if the pid file exists, with the PID in it (non-zero size)
-    if [ -s "${PIDFILE}" ]; then
-        log_success_msg $"${prog} startup"
-        echo
-        touch "${lockfile}"
-        return 0
-    else
-        log_failure_msg $"${prog} startup"
-        echo
-        return 1
-    fi
+    return $retval
 }
 
 stop() {
     echo -n $"Stopping ${prog}: "
-    if [ ! -s "${PIDFILE}" ] ; then
-        print_service_pid > "${PIDFILE}"
-    fi
-
-    killproc -p "${PIDFILE}" -t"${SERVICE_STOP_RETRIES}"s -- "${JAVA_BIN}" "-XX:OnOutOfMemoryError=\"kill -9 %p\"" "${JAVA_ARGS}"
+    "${INSTALL_DIR}/bin/${realname}" stop
     rc_status -v
     retval=$?
 
-    if [ "$retval" = 0 ]; then
```

Comment:
Same rationale for the seemingly redundant status message line here as with the `start` command.